### PR TITLE
Create join table for index and choices #47

### LINKED ISSUE #47: Create join table for index and choices

### PR DESCRIPTION
**Pull request by grimm**
FIxes #47

### CODE REVIEW

File: test/fixtures/index_choices.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  index_id: 1
+  choice_id: 1
```
Comment (teknofire) [inline test/fixtures/index_choices.yml]: these fixtures are invalid, they should be using 

```
one:
  index: one
  choice: one
```

and then the second one should probably reference choise two

**Comment by grimm** [comment]
@gina-alaska/awrvi Fixed

**Comment by teknofire** [comment]
The fixtures still aren't right, don't have the _id and it's not a symbol just the word one

**Comment by grimm** [comment]
@gina-alaska/awrvi Fixed Again

File: db/migrate/20160115012332_create_index_choices.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateIndexChoices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :index_choices do |t|
+      t.integer :index_id
+      t.integer :choice_id
```
Comment (teknofire) [inline db/migrate/20160115012332_create_index_choices.rb]: make sure we to use the references helpers for migration

**Comment by grimm** [comment]
@gina-alaska/awrvi Ready?

**Comment by teknofire** [comment]
Still need the fix for the index_choices model/migration to use references migration stuff

**Comment by grimm** [comment]
@gina-alaska/awrvi Ready?

**Comment by teknofire** [comment]
:+1:

**Comment by smacfarlane** [comment]
:+1:

**Pull request merged**